src/tests/ftest/cart/corpc/cart_corpc_five_node.py

```diff
@@ -48,8 +48,10 @@ def setUp(self):
 
     def tearDown(self):
         """ Tear down """
+        self.report_timeout()
+        self._teardown_errors = []
```

Comment:
This shouldn't be needed. The list is initialized [here](https://github.com/daos-stack/daos/blob/master/src/tests/ftest/util/apricot/apricot/test.py#L141).

Reply:
54ef358b6e0da1a615e47e6acaf71d5b5255d59a